src/validate.py

```diff
@@ -26,6 +26,11 @@ def decorated_function(*args, **kwargs):
         year = kwargs.get('year')
         chapter = kwargs.get('chapter')
 
+        # Handle the pages that don't follow /lang/year/page structure
+        # which can end up here if they have trailing slashes in URL
+        if year == "accessibility-statement":
+            return redirect('/%s/accessibility-statement' % lang, code=301)
```

Comment:
Not sure that this is a job for the validator. Could this be avoided by adding

```
@app.route('/<lang>/accessibility-statement/')
```

to the AS renderer and redirecting as needed?

Reply:
Done!

Note we can't do this for the chapters.